### PR TITLE
#46 - Add CORS support

### LINKED ISSUE #46: Add CORS support
Example:
https://pypi.org/project/aiohttp_cors/
https://docs.aiohttp.org/en/v0.20.0/web.html#middlewares
https://aiohttp-middlewares.readthedocs.io/en/latest/_modules/aiohttp_middlewares/cors.html

### LINKED ISSUE #49: Update to latest `api_gateway_common` version

### PR DESCRIPTION
**Pull request by vladyslav-fenchak**

### CODE REVIEW

File: minos/api_gateway/rest/service.py [hunk 1]
```diff
@@ -25,7 +28,12 @@ def __init__(self, address: str, port: int, config: MinosConfig):
         super().__init__(address, port)
 
     async def create_application(self) -> web.Application:
-        app = web.Application()
+        middlewares = []
+        if self.config.cors.enabled:
+            middlewares = [cors_middleware(allow_all=self.config.cors.enabled)]
```
Comment (garciparedes) [inline minos/api_gateway/rest/service.py]: As you've already checked that the `self.config.cors.enabled` is `True`, it's preferable to directly pass the value (for readability reasons).
```suggestion
            middlewares = [cors_middleware(allow_all=True)]
```

File: minos/api_gateway/rest/service.py [hunk 2]
```diff
@@ -25,7 +28,12 @@ def __init__(self, address: str, port: int, config: MinosConfig):
         super().__init__(address, port)
 
     async def create_application(self) -> web.Application:
-        app = web.Application()
+        middlewares = []
```
Comment (garciparedes) [inline minos/api_gateway/rest/service.py]: When empty lists are created, it's preferable to directly call `list()`
```suggestion
        middlewares = list()
```

**Review by garciparedes — APPROVED** [review]
Hi @vladyslav-fenchak, awesome changes! 🎉  Could you apply my suggestions before merging it?

File: minos/api_gateway/rest/service.py (hunk 2 above)
Comment (vladyslav-fenchak) [reply minos/api_gateway/rest/service.py]: Done!

File: minos/api_gateway/rest/service.py (hunk 1 above)
Comment (vladyslav-fenchak) [reply minos/api_gateway/rest/service.py]: Done!

**Pull request merged**